src/LLVM_Runtime_Linker.cpp

```diff
@@ -1087,6 +1089,7 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
                 modules.push_back(get_initmod_x86_avx512_ll(c));
             }
             if (t.has_feature(Target::AVX512_SapphireRapids)) {
+                modules.push_back(get_initmod_amx(c, bits_64, debug));
```

Comment:
This appears to have the possibility of injecting Linux specific code on non-Linux paltforms.